spec/features/admin_claim_allocation_spec.rb

```diff
@@ -4,12 +4,28 @@
   before do
     submitted_claims = []
     @signed_in_user = sign_in_as_service_operator
+
+    # index: 0-1
     submitted_claims << create_list(:claim, 2, :submitted, policy: StudentLoans)
+
+    # index: 2-14
```

Comment:
Don't worry about it now but I have no idea why the original author (not me) chose such strange random magic numbers in these specs